shopping-cart/app/com/commercetools/sunrise/shoppingcart/common/SunriseFrameworkCartController.java

```diff
@@ -117,14 +114,15 @@ private void postInit() {
 
     protected CompletionStage<Cart> applySideEffects(@Nonnull Cart cart) {
         final Http.Session session = session();
-        return updateCartWithUserPreferences(cart, userContext()).thenApply(updatedCart -> {
-            CartSessionUtils.overwriteCartSessionData(updatedCart, session, userContext(), productReverseRouter);
-            return updatedCart;
-        })
+        return updateCartWithUserPreferences(cart, userContext())
                 .thenApply(updatedCart -> {
+                    final MiniCartBeanFactory miniCartBeanFactory = injector().getInstance(MiniCartBeanFactory.class);
```

Comment:
This is actually the only thing added here, the rest it's just a bit of (involuntary) formatting.